extension/scripts/features/stacking-mode/ttStackingMode.js

```diff
@@ -0,0 +1,136 @@
+"use strict";
+
+(async () => {
+	if (!getPageStatus().access) return;
```

Comment:
Since mini profiles are included, this should not be blocked by the page access.